lib/ansible/modules/windows/win_disk_facts.ps1

```diff
@@ -0,0 +1,132 @@
+#!powershell
+
+# Copyright: (c) 2017, Marc Tschapek <marc.tschapek@itelligence.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+$ErrorActionPreference = "Stop"
+Set-StrictMode -Version 2.0
+
+# Create a new result object
+$result = @{
+    changed = $false
+    ansible_facts = @{
+        ansible_disk = @{
+        } 
+    }
+}
+
+# Search disks
+try {
+    $disk = Get-Disk
```

Comment:
I would check to make sure `Get-Disk is available on the OS, IIRC it was added in Server 2012/Windows 8.

See https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/windows/win_region.ps1#L250 on how this can be done in Powershell. If the cmdlet isn't available we should fail with an exception message that tells us why.

Reply:
Implemented